recipe-server/normandy/recipes/management/commands/update_actions.py

```diff
@@ -57,14 +58,26 @@ def handle(self, *args, **options):
 
 
 def get_implementation(action_name):
+    if action_name in settings.REMOTE_ACTIONS:
+        # Remote actions don't have an implementation since their
+        # implementation is stored in mozilla-central and not available as part of the repo.
+        return
     chunks = get_loader('ACTIONS').get_assets()['chunks']
     implementation_path = chunks[action_name][0]['path']
     with open(implementation_path) as f:
         return f.read()
 
 
 def get_arguments_schema(action_name):
-    action_directory = settings.ACTIONS[action_name]
-    with open(os.path.join(action_directory, 'package.json')) as f:
-        action_metadata = json.load(f)
-        return action_metadata['normandy']['argumentsSchema']
+    if action_name in settings.REMOTE_ACTIONS:
```

Comment:
Note the order. If an action (e.g. `console-log`) appears in *both* `settings.ACTIONS` and `settings.REMOTE_ACTIONS`, then the first (and only) place it will look for the argument schema is via `settings.REMOTE_ACTIONS`. Makes sense. It's the future.